components/relay/src/lib.rs

```diff
@@ -13,12 +13,33 @@ use serde::{Deserialize, Serialize};
 use url::Url;
 use viaduct::{header_names, Method, Request};
 
+/// Represents a client for the Relay API.
+///
+/// Use this struct to connect and authenticate with a Relay server,
+/// managing authorization to call protected endpoints.
+///
+/// # Authorization
+/// - Clients should user the `FirefoxAccount.getAccessToken()` function
```

Comment:
I have no idea if/how any of this works in terms of rustdocs etc 😅 - but maybe  `fxa_client::FirefoxAccount::getAccessToken()`? I doubt either of these will render a workable link though.